src/module/item/class/sheet.ts

```diff
@@ -11,7 +11,10 @@ export class ClassSheetPF2e extends ABCSheetPF2e<ClassPF2e> {
         return {
             ...sheetData,
             skills: CONFIG.PF2E.skills,
-            proficiencyChoices: CONFIG.PF2E.proficiencyLevels,
+            proficiencyChoices: CONFIG.PF2E.proficiencyLevels.map((p, index) => ({
```

Comment:
As of the newest release, we no longer have to do this.

Reply:
yeah, that works now.